tests/Aspire.Microsoft.EntityFrameworkCore.Cosmos.Tests/AspireAzureEfCoreCosmosDBExtensionsTests.cs

```diff
@@ -22,15 +22,11 @@ public void CanConfigureDbContextOptions()
         builder.Configuration.AddInMemoryCollection([
             new KeyValuePair<string, string?>("ConnectionStrings:cosmosConnection", ConnectionString),
             new KeyValuePair<string, string?>("Aspire:Microsoft:EntityFrameworkCore:Cosmos:Region", "westus"),
+            new KeyValuePair<string, string?>("Aspire:Microsoft:EntityFrameworkCore:Cosmos:RequestTimeout", "608"),
         ]);
 
-        builder.AddCosmosDbContext<TestDbContext>("cosmosConnection", "databaseName", configureDbContextOptions: optionsBuilder =>
-        {
-            optionsBuilder.UseCosmos(ConnectionString, "databaseName", cosmosBuilder =>
-            {
-                cosmosBuilder.RequestTimeout(TimeSpan.FromSeconds(608));
```

Comment:
I don't think this test should be changed like this. It's intention is that you can configure the DbContextOptions object in code. It ensures that you can configure things in configuration and in code and they are both respected. With this change we lost the verification that changing options in code are respected.

Reply:
This applies to all the below tests as well.